import-scripts/update-az-mskimpact.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+export AZ_DATA_HOME=$PORTAL_DATA_HOME/az-msk-impact-2022
+export AZ_MSK_IMPACT_DATA_HOME=$AZ_DATA_HOME/mskimpact
+export AZ_TMPDIR=$AZ_DATA_HOME/tmp
+
+source $PORTAL_HOME/scripts/dmp-import-vars-functions.sh
+
+# 1. Pull latest from AstraZeneca repo (mskcc/az-msk-impact-2022)
+printTimeStampedDataProcessingStepMessage "pull of AstraZeneca data updates to git repository"
+
+cd $AZ_DATA_HOME ; $GIT_BINARY pull origin
+
+if [ $? -gt 0 ] ; then
+    sendPreImportFailureMessageMskPipelineLogsSlack "GIT PULL (az-msk-impact-2022) :fire: - address ASAP!"
+
+    EMAIL_BODY="Failed to pull AstraZeneca incoming changes from Git - address ASAP!"
+    echo -e "Sending email $EMAIL_BODY"
+    echo -e "$EMAIL_BODY" | mail -s "[URGENT] GIT PULL FAILURE" $PIPELINES_EMAIL_LIST
+
+    exit 1
+fi
+
+# ------------------------------------------------------------------------------------------------------------------------
+# 2. Copy data from local clone of MSK Solid Heme repo to local clone of AZ repo
+
+# Create temporary directory to store data before subsetting
+if ! [ -d "$AZ_TMPDIR" ] ; then
+    if ! mkdir -p "$AZ_TMPDIR" ; then
+        echo "Error : could not create tmp directory '$AZ_TMPDIR'" >&2
+        exit 1
+    fi
+fi
+if [[ -d "$AZ_TMPDIR" && "$AZ_TMPDIR" != "/" ]] ; then
+    rm -rf "$AZ_TMPDIR"/*
+fi
+
+cp -r $MSK_SOLID_HEME_DATA_HOME/* $AZ_TMPDIR
```

Comment:
if you add "-a" to the cp command, it preserves file metadata ... such as creation times, last edit time, file permissions, etc..